tooling/bundler/src/bundle/windows/msi/wix.rs

```diff
@@ -176,37 +176,18 @@ fn copy_icon(settings: &Settings, filename: &str, path: &Path) -> crate::Result<
 fn app_installer_output_path(
   settings: &Settings,
   language: &str,
-  version: &str,
   updater: bool,
 ) -> crate::Result<PathBuf> {
-  let arch = match settings.binary_arch() {
-    "x86" => "x86",
-    "x86_64" => "x64",
-    "aarch64" => "arm64",
-    target => {
-      return Err(crate::Error::ArchError(format!(
-        "Unsupported architecture: {}",
-        target
-      )))
-    }
-  };
-
-  let package_base_name = format!(
-    "{}_{}_{}_{}",
-    settings.product_name(),
-    version,
-    arch,
-    language,
-  );
+  let msi_name =
+    crate::bundle::bundle_name_with_suffix(settings, &language.replace('-', "_"), "msi");
```

Comment:
well that's true, but with this change it's also the separator for the parts in the bundle file name.. I'd prefer using `_` instead, but some people prefer `-`

Reply:
there will always people loving it the other way around, I don't see an issue with `App-1.2.0-x64-en-US.exe`

Reply:
> there will always people wanting it the other way around but I don't see an issue with `App-1.2.0-x64-en-US.exe`, seems better than `App-1.2.0-x64-en_US.exe`

I think this is about consistency rather than standards. The hyphen is a separator between different parts of the filename, while the underscore is a separator between words within a part, such as Snow_White-v1.0.1_beta1-x86_64-en_US.deb. Under the consistency naming convention, it is easy to parse the various standard-compliant parts.

Reply:
> there will always people wanting it the other way around but I don't see an issue with `App-1.2.0-x64-en-US.exe`, seems better than `App-1.2.0-x64-en_US.exe`

I think this is about consistency rather than standards. The hyphen(-) is a separator between different parts of the filename, while the underscore(_) is a separator between words within a part, such as Snow_White-v1.0.1_beta1-x86_64-en_US.deb. Under the consistency naming convention, it is easy to parse the various standard-compliant parts.